names.txt

```diff
@@ -7,3 +7,4 @@ ALi
 Sina
 arman
 Alireza
+Sadegh Mhz
```

Comment:
To merge your pull request, you just need to add your first name to the end of the file!